src/locales/en.json

```diff
@@ -640,6 +640,16 @@
     "by-nc-sa": "BY-NC-SA",
     "by-nc-nd": "BY-NC-ND"
   },
+  "license-readable-names": {
+    "cc0": "Zero",
+    "pdm": "Public Domain Mark",
+    "by": "Attribution",
+    "by-sa": "Attribution ShareAlike",
```

Comment:
I wonder what the best solution is for the "readable" names. I know these are the official names, but since they are used for the screenreader, maybe it's better to split the 'ShareAlike'? Because at the moment, it is read as 'sharelike', and does not give screenreader users the most accurate representation...

Reply:
That is an important question here. I was wondering the same thing and whether to add "CC" at the beginning and/or "license" at the end to be more descriptive. Does it change the meaning significantly if we split "ShareAlike"? I'm not the best person to answer this so I'll apply what the rest suggest. CC @sarayourfriend

Reply:
https://github.com/WordPress/openverse-frontend/pull/422#issuecomment-974043628

VoiceOver reads 'Attribution-ShareAlike' correctly for me, but I think it would also read 'Share-Alike' correctly if we wanted to add a hyphen there. Since these strings are being _heard_, not seen, I think how it sounds is more important and it's okay to change the spacing/hyphenation as-needed.